### PR TITLE
[origin-ci] Gce tweaks for origin-ci

### PR DESCRIPTION
**Pull request by detiber**

### CODE REVIEW

File: reference-architecture/gce-ansible/playbooks/openshift-install.yaml [hunk 1]
```diff
@@ -30,6 +28,8 @@
         - service: master http proxy
           port: 8080/tcp
 
+- include: openshift-setup.yaml
+
```
Comment (detiber) [inline reference-architecture/gce-ansible/playbooks/openshift-install.yaml]: @smarterclayton creating the proxy instance service before including openshift-setup.yaml allows for the master lbs to be functional, and prevents issues with node provisioning and other api-related tasks (router, registry, etc).

File: reference-architecture/gce-ansible/playbooks/roles/master-http-proxy/tasks/main.yaml [hunk 2]
```diff
@@ -10,6 +10,8 @@
       [Service]
       Type=simple
       ExecStart=/usr/bin/kubectl proxy -p 8080 --address=0.0.0.0 --accept-hosts=^*$ --config=/etc/origin/master/admin.kubeconfig
+      Restart=always
+      RestartSec=5s
```
Comment (detiber) [inline reference-architecture/gce-ansible/playbooks/roles/master-http-proxy/tasks/main.yaml]: Ensure that the proxy restarts on failure

File: reference-architecture/gce-cli/gcloud-provision.sh [hunk 3]
```diff
@@ -319,8 +319,21 @@ if [[ -n "${STARTUP_SCRIPT_FILE:-}" ]]; then
         echo "Startup script file missing at ${STARTUP_SCRIPT_FILE} - create an empty file, run ./gcloud-startup.sh, or unset STARTUP_SCRIPT_FILE"
         exit 1
     fi
-    metadata+="--metadata-file=startup-script=${STARTUP_SCRIPT_FILE} "
+    metadata+="--metadata-from-file=startup-script=${STARTUP_SCRIPT_FILE}"
 fi
+if [[ -n "${USER_DATA_FILE:-}" ]]; then
```
Comment (detiber) [inline reference-architecture/gce-cli/gcloud-provision.sh]: Add ability to set user-data

File: reference-architecture/gce-cli/user-data [hunk 4]
```diff
@@ -0,0 +1,8 @@
+#cloud-config
+
+write_files:
+- path: /etc/sudoers.d/99-openshift-cloud-init-requiretty
+  owner: 'root:root'
+  permissions: '0440'
+  content: |
+    Defaults:cloud-user !requiretty
```
Comment (detiber) [inline reference-architecture/gce-cli/user-data]: disables requiretty without having to bake it into the image.

**Comment by smarterclayton** [comment]
Did these merge into master as well?

**Pull request closed without merging**